tools/syz-crush/crush.go

```diff
@@ -24,107 +28,252 @@ import (
 )
 
 var (
-	flagConfig = flag.String("config", "", "configuration file")
+	flagConfig      = flag.String("config", "", "configuration file")
+	flagRestartTime = flag.Duration("restartTime", 0, "restartPeriod how long to run the test.")
+	flagInfinite    = flag.Bool("infinite", true, "by default test is run for ever. -infinite=false to stop on crash")
 )
 
+type CrashReport struct {
```

Comment:
Introducing a type for this looks like over-complication.
We can log vm number and crash title right in runInstance, and then the index is not used for anything useful (except for producing more duplicate log lines about the crash we already logged in runInstance).